zipkin/src/test/java/com/linecorp/armeria/client/tracing/HttpTracingClientTest.java

```diff
@@ -108,12 +106,13 @@ public void shouldSubmitSpanWhenSampled() throws Exception {
         assertThat(span.annotations()).hasSize(2);
 
         // check tags
-        assertThat(span.tags()).containsAllEntriesOf(ImmutableMap.of(
-                "http.host", "foo.com",
-                "http.method", "POST",
-                "http.path", "/hello/armeria",
-                "http.status_code", "200",
-                "http.url", "none+h2c://foo.com/hello/armeria"));
+        assertThat(span.tags()).containsEntry("http.host", "foo.com")
+                               .containsEntry("http.method", "POST")
+                               .containsEntry("http.path", "/hello/armeria")
+                               .containsEntry("http.status_code", "200")
+                               .containsEntry("http.url", "http://foo.com/hello/armeria")
+                               .containsEntry("http.protocol", "h2c")
+                               .containsEntry("http.serfmt", "none");
```

Comment:
as this creates index load, possibly add when not none?

Reply:
Updated.